resources/istio/templates/configmap-extra-manifests.yaml

```diff
@@ -5,4 +5,8 @@ metadata:
   name: kyma-extra-manifests
 data:
   destination-rules.yaml: |-
-{{tpl (.Files.Get "files/destination-rules.yaml" | printf "%s" | indent 4) .}}
+{{- if .Values.global.isLocalEnv }}
+{{ tpl (.Files.Get "files/destination-rules-minikube.yaml" | printf "%s" | indent 4) . }}
+{{- else }}
```

Comment:
and then this if else can be simplified to single if containing additional DR for ingress :)